src/components/DetailModal/index.js

```diff
@@ -0,0 +1,109 @@
+            <p class="bg-success text-white rounded-pill px-1"><strong>{data.type}</strong></p>
+            </div>
+            <div class="row mx-auto">
+                <img src={data.logo} class="img-thumbnail col-1" />
+                <p><strong>{data.company}</strong></p>
+            </div>
+            <button type="button" className="btn btn-primary" data-bs-toggle="modal" data-bs-target="#exampleModal">
+                Launch demo modal
+            </button>
+
+            <div className="modal fade" id="exampleModal" tabIndex="-1" aria-labelledby="exampleModalLabel"
+                 aria-hidden="true">
+                <div className="modal-dialog modal-lg">
+                    <div className="modal-content">
+                        <div className="modal-header">
+                            <p className="modal-title fs-6" id="exampleModalLabel">
+                                <strong>{data.job_title} - {data.company}</strong></p>
+                            <button type="button" className="btn-close" data-bs-dismiss="modal"
+                                    aria-label="Close"></button>
+                        </div>
+                        <div className="modal-body">
+                            <div className="container">
+
+                                <div className="d-flex flex-row bg-primary text-white py-5 fs-4">
+                                    <p className="mx-4"><strong>{data.job_title}</strong></p>
+                                    <p className="bg-success text-white rounded-pill px-3"><strong>{data.type}</strong>
+                                    </p>
+                                </div>
+                                <div class="border border-5 border-light p-3">
+                                <div className="row mx-auto" pt-3>
+                                    <img src={data.logo} className="col-2"/>
+                                    <p class="col-3 align-self-center"><strong>{data.company}</strong></p>
+                                </div>
+
+                                <h2 class="fs-5 pt-2">Key facts:</h2>
+                                    <table class="table table-borderless table-sm">
+
+                                    <thead>
+                                    <tr>
+                                        <td scope="col">Salary:</td>
+                                        <td scope="col">Type:</td>
+                                        <td scope="col">Date posted:</td>
+                                        <td scope="col">Skills:</td>
+                                    </tr>
+                                    </thead>
+                                    <tbody>
+                                    <tr>
+                                        <td>£{data.salary ?? "Not specified"}</td>
+                                        <td>{data.type ?? "Not specified"}</td>
+                                        <td>{data.posted}</td>
```

Comment:
Posted needs to be converted from International date format into UK date

Reply:
Done!